plugins/techdocs/src/reader/components/TechDocsReaderPage.tsx

```diff
@@ -57,20 +57,21 @@ export const TechDocsReaderPage = (props: TechDocsReaderPageProps) => {
 
   const [documentReady, setDocumentReady] = useState<boolean>(false);
   const { namespace, kind, name } = useParams();
+  const doc = { kind: kind!, namespace: namespace!, name: name! };
```

Comment:
could you name these `entityRef` instead of `doc`?

Reply:
definitely!